pages/plugins/using.md

```diff
@@ -47,8 +47,7 @@ location.
 
 ## Configuring plugins
 
-Plugins are configured using attributes in your pipeline YAML configuration. The simplest plugin is one that accepts no configuration, such as the [Library Example plugin](https://github.com/buildkite-plugins/library-example-buildkite-plugin):
-
+Plugins are configured using attributes on steps in your pipeline YAML configuration. While you can't define plugins at a pipeline level, you can use [YAML anchors](https://github.com/buildkite/docs/pull/1968#using-yaml-anchors-with-plugins) to avoid repeating the plugin code over multiple steps. The simplest plugin is one that accepts no configuration, such as the [Library Example plugin](https://github.com/buildkite-plugins/library-example-buildkite-plugin):
```

Comment:
Mainly the docs link that needs updating, but while we're here, let's avoid the repetition in the first sentence:

```suggestion
Plugins are configured using attributes on steps in your pipeline YAML definition. While you can't define plugins at a pipeline level, you can use [YAML anchors](/docs/plugins/using#using-yaml-anchors-with-plugins) to avoid repeating the plugin code over multiple steps. The simplest plugin is one that accepts no configuration, such as the [Library Example plugin](https://github.com/buildkite-plugins/library-example-buildkite-plugin):
```